website/src/word-panel.css.tsx

```diff
@@ -0,0 +1,39 @@
+import { style } from "@vanilla-extract/css"
+import sprinkles, { hspace, mediaQueries, theme, vspace } from "./sprinkles.css"
+
+export const wholePanel = style({
+  position: "sticky",
+  top: 125,
+})
+
+export const wordPanelButton = style({
```

Comment:
You can look at the close button for the `MorphemeDialog` which already has some spacing styles & clears out the background so it's just the icon. I think that styling will look better here.